requirements.txt

```diff
@@ -27,7 +30,7 @@ jsonschema==2.5.1  # MIT
 lxml==3.6.0  # BSD
 nameparser==0.4.0  # LGPL
 newrelic==2.68.0.50  # newrelic APM agent, Custom License
-pendulum==0.6.2  # MIT
+pendulum==1.2.0  # MIT
```

Comment:
Ack. That's not great. I'll take a look at all the places we're using pendulum and update accordingly. It's probably better to rely on version 1.X.